packages/bpk-mixins/src/mixins/_images.scss

```diff
@@ -33,8 +33,14 @@
   position: relative;
   display: block;
   max-width: 100%;
+  transition: background-color $bpk-duration-base ease-in-out;
   background-color: $bpk-color-gray-100;
   overflow: hidden;
+  transition-delay: $bpk-duration-base;
+}
+
+@mixin bpk-image--show {
```

Comment:
This doesn't need to be a mixin - just have it inline. In fact the mixins for the image component are realistically never going to be used by consumers. We should remove them (perhaps outside the scope of this PR).

Reply:
Done here https://github.com/Skyscanner/backpack/pull/82